lua/glide/server/weaponry.lua

```diff
@@ -93,12 +93,12 @@ do
         traceData.endpos = pos + dir * length
         traceData.filter = traceFilter
 
-        local tr = TraceLine( traceData )
+        local tr = TraceHull( traceData )
```

Comment:
I think the mins/maxs were not set before this point, tbh I'm not sure if that's an issue